src/iop/colorzones.c

```diff
@@ -568,12 +569,19 @@ static void dt_iop_colorzones_get_params(dt_iop_colorzones_params_t *p, const in
   }
 }
 
+static void picker_scale(const float *const in, float *out)
```

Comment:
We have a picker_scale in tonecurve.c, the implementation is quite different even if somehow similar. Is that expected? Is that because we have two different CS?